src/BlockchainReader/Bitcoin.test.ts

```diff
@@ -0,0 +1,110 @@
+import { PoetTimestamp } from '@po.et/poet-js'
+import { allPass, equals } from 'ramda'
+import { describe } from 'riteway'
+
+import { PREFIX_POET, PREFIX_BARD } from 'Helpers/Bitcoin'
+
+import { blockToPoetAnchors } from './Bitcoin'
+
+import * as TestBlock from './TestData/block-00000000000151360aad32397ff1cf7dd303bed163b0ef425e71a53ccdec7312.json'
+
+describe('Bitcoin.blockToPoetAnchors', async should => {
+  const { assert } = should()
+
+  assert({
+    given: 'testnet block 00000000000151360aad32397ff1cf7dd303bed163b0ef425e71a53ccdec7312',
+    should: 'satisfy basic integrity checks',
+    actual: validateTestBlockIntegrity(TestBlock),
+    expected: true,
+  })
+
+  const ipfsDirectoryHashes = [
+    'QmSGQKnfG98KrpxNpZMhNyAKkvxudGqKhGeGv13zSXLQwz',
+    'QmSicKkyyb5NJqSJ9EaaMJWQvqa4e3CX7psjjnRxvgfodv',
+    'QmTrtzm1fvysZgsGhJicTrdJ1vSbi3UuLWBiHAMYBkcQfL',
+    'QmaXCvSA4noYsJruubE8cYUtu3gPAmgL9aosFzDdrsviWJ',
+    'QmYMHmt9H37gqwDMd4yYrt99cDRJxHpwVATKWYGbYNWncp',
+  ]
+
+  {
+    const poetAnchors = blockToPoetAnchors(TestBlock as any) // as any: see footer note
+
+    const isBardAnchor = (poetAnchor: PoetTimestamp) => poetAnchor.prefix === PREFIX_BARD
+    const anchorIsVersion0003 = (poetAnchor: PoetTimestamp) => equals(poetAnchor.version, [0, 0, 0, 3])
+    const anchorIsBlockHash = (poetAnchor: PoetTimestamp) => poetAnchor.blockHash === TestBlock.hash
+    const anchorIsBlockHeight = (poetAnchor: PoetTimestamp) => poetAnchor.blockHeight === TestBlock.height
+    const anchorHasUnexpectedIpfsHash = (poetAnchor: PoetTimestamp) =>
+      !ipfsDirectoryHashes.includes(poetAnchor.ipfsDirectoryHash)
+
+    const anchorsIpfsHashes = poetAnchors.map(poetAnchor => poetAnchor.ipfsDirectoryHash)
+
+    assert({
+      given: 'blockToPoetAnchors(TestBlock)',
```

Comment:
These should read like descriptions. Something like:
- given:' a test block with poet anchors'

Reply:
@warrenv does [this commit](https://github.com/poetapp/node/pull/383/commits/8790c3a466fa46fdab35fe90923e97ecf56af1ee) fix it?